src/arrays.js

```diff
@@ -1,61 +1,76 @@
 const getNthElement = (index, array) => {
-  // your code here
+  m = (array.length);
+  if(index>0){
+    index = index % m;
+  };
+  return array[index];
```

Comment:
There is a one line solution for this exercise. You have got all the correct logic, so do you think you could refactor it to one line? A hint is you just need to add some logic to the code you already have on line 6. :)